src/utils/getMatchedPassagesFromFamiy.ts

```diff
@@ -0,0 +1,15 @@
+import { TDocumentPage, TMatchedFamily, TPassage } from "@types";
+
+export const getMatchedPassagesFromSearch = (families: TMatchedFamily[], document: TDocumentPage): [TPassage[], number] => {
+  let passageMatches: TPassage[] = [];
+  let totalNoOfMatches = 0;
+  families.forEach((family) => {
+    family.family_documents.forEach((cacheDoc) => {
```

Comment:
question: why is this called `cacheDoc`?

Reply:
Not sure, renamed.